lib/nolibc/README.md

```diff
@@ -0,0 +1,245 @@
+# nolibc, Unikraft's Lightweight Subset of the C Standard Library
+
+nolibc provides the basic core functionalities of libc in a lighter
+and more compact package, fulfilling the needs of most simple applications
+without occupying a large amount of memory, while also remaining POSIX-compliant.
+More complex applications that require a more comprehensive set of
+standard C
+functions are better suited with Unikraft's port of
+[musl](https://github.com/unikraft/lib-musl).
+
+# Components
+
+nolibc exposes the following symbols for external use:
+
+<details>
```

Comment:
I would try to avoid using inline html as much as possible, you can just do a markdown list using `*` and add links to the man pages.
```md
* [function-name](link-to-man-page)
```